### ISSUE TITLE
How can I set weight decay when using digits

### ISSUE THREAD

**Issue by Seakia** [NVIDIA/DIGITS#1687]
Hi, I want to change weight decay to avoid overfitting, how can I set it? I don't find the corresponding item when I create one model, such as classification.

Thanks.
